test/csuite/timestamp_abort/main.c

```diff
@@ -131,7 +131,7 @@ extern char *__wt_optarg;
     "M,create,"                                               \
     "debug_mode=(table_logging=true,checkpoint_retention=5)," \
     "eviction_updates_target=20,eviction_updates_trigger=90," \
-    "log=(enabled,file_max=10M,remove=true),session_max=%d,"  \
+    "log=(enabled,file_max=10M,remove=false),session_max=%d," \
```

Comment:
Ah, thanks for catching that—that was committed by mistake! `remove=false` is needed for the debug log to be complete, because WiredTiger otherwise will start removing old log files.

(When we incorporate model-based verification to `timestamp_abort`, we will need to set `remove` to `false`, but at that point, I will do it conditionally only if we are actually going to need this. Otherwise this will produce many GBs of logs.)